src/main/resources/edu/cornell/kfs/paymentworks/document/datadictionary/PaymentWorksFieldMappingMaintenanceDocument.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+    <bean id="PaymentWorksFieldMappingMaintenanceDocument" parent="PaymentWorksFieldMappingMaintenanceDocument-parentBean"/>
+    <bean id="PaymentWorksFieldMappingMaintenanceDocument-parentBean" abstract="true" parent="MaintenanceDocumentEntry">
+        <property name="businessObjectClass" value="edu.cornell.kfs.paymentworks.businessobject.PaymentWorksFieldMapping"/>
+        <property name="maintainableClass" value="org.kuali.kfs.kns.maintenance.KualiMaintainableImpl" />
```

Comment:
I'm wondering if org.kuali.kfs.sys.document.FinancialSystemMaintainable might be more appropriate here.